code/bignum/Hacl.Bignum.Convert.fst

```diff
@@ -189,6 +189,10 @@ val mk_bn_to_bytes_be:
   -> len:size_t{0 < v len /\ numbytes t * v (blocks len (size (numbytes t))) <= max_size_t} ->
   bn_to_bytes_be_st t len
 
+(* This causes the verification conditition for the continuation
+of the call to this function to be done in a separate Z3 query. *)
+let break () : STATE unit (fun p h -> spinoff (squash (p () h))) = ()
+
```

Comment:
should this nice trick go into a separate file so as to be reusable? is this generally applicable?

Reply:
Separate file sounds good! Just not sure where in the repo to place it.

It's *mostly* generally applicable, it's really just about wrapping the VC with `spinoff`. But for Pure/Lemma, it gets very hairy due to monotonicity of the WPs (https://github.com/FStarLang/FStar/pull/3174), so I want to get that into F* itself.

Reply:
yeah just wondering if this needs to go in lib or something

but maybe we'll just wait for it to become an official thing in F*